### PR TITLE
add the Department class, set the primary Key on it and set the forei…

### PR DESCRIPTION
**Pull request by MauJho**
…ng key on the User class

### CODE REVIEW

**Review by HeyBaldur — APPROVED** [review]

File: Api Web/Models/Department.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Api_Web.Models
+{
+    public class Department
+    {
+        [Key]
+        public int DepartmentEmployeeId { get; set; }
```
Comment (HeyBaldur) [inline Api Web/Models/Department.cs:12]: everything seems to be in good shape however we could change the variable to Id, or EmployeeId, something more understandable for other devs.

File: Api Web/Models/Department.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        public string Departments{ get; set; }
```
Comment (HeyBaldur) [inline Api Web/Models/Department.cs:13]: If we say departments, we expect to have many, but the variable is one single string, what do we expect to receive here?

File: Api Web/User.cs [hunk 3]
```diff
@@ -18,6 +19,13 @@ public class User
 
         public string Email { get; set; }
 
+        public int EmployeeId { get; set; }
+
         public DateTime DateOfBirth { get; set; }
+
+        //foreing key for Departments
+        public int DepartmentEmployeeId { get; set; }
```
Comment (HeyBaldur) [inline Api Web/User.cs:27]: We could name it such as EmployeeId

File: Api Web/User.cs [hunk 4]
```diff
@@ -18,6 +19,13 @@ public class User
...
+
+        public Department Departments { get; set; }
```
Comment (HeyBaldur) [inline Api Web/User.cs:29]: Same here, Department != Departments, we must be consistent with names that make sense.

**Review by HeyBaldur — CHANGES_REQUESTED** [review]

File: Api Web/Models/Department.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Api_Web.Models
+{
+    public class Department
+    {
+        [Key]
```
Comment (HeyBaldur) [inline Api Web/Models/Department.cs:11]: If we add only the Id, we don't need the attribute key

Comment (MauJho) [reply Api Web/Models/Department.cs]: Changes has been done, please check. @HeyBaldur

**Review by HeyBaldur — APPROVED** [review]

**Pull request merged**